pkg/catalog/loader/loader.go

```diff
@@ -33,6 +33,7 @@ import (
 const (
 	httpPrefix  = "http://"
 	httpsPrefix = "https://"
+	AuthStoreId = "auth_store"
```

Comment:
Do we need to export this?

Reply:
Auth store created at [different pkg ](https://github.com/projectdiscovery/nuclei/pull/5646/files#diff-ff41ea4f2278e307fbd5fb6d33ed0f14bf10b15307700ae206bf79882dd04b44R54) and it's comparison happens at [different pkg](https://github.com/projectdiscovery/nuclei/pull/5646/files#diff-ba6b81be37a7c7fc3c03e1c11f29834dda5f3e7603968e507fc59eebd06a2b16R484) so thought to expose it, any other suggestions?